L26-Ability-to-restrict-max-threads-in-C++SyncServer.md

```diff
@@ -0,0 +1,93 @@
+* Author(s): Sree Kuchibhotla (sreecha)
+* Approver: vjpai
+* Status: In Review
+* Implemented in: C++
+* Last updated: June 6, 2018
+* Discussion at: 
```

Comment:
Can you please start a discussion thread on grpc-io about this and add the URL? Most of the technical questions should actually belong there.

Reply:
Done.